### ISSUE TITLE
Ensure the schedule notice in deposits list shows consistent info as Payments Overview summary

### ISSUE THREAD

**Issue by Jinksi** [Automattic/woocommerce-payments#8026]
### Description

In #7883 we shipped a notice which was intended to be a quick and temporary solution to #7887.

<img width="2047" alt="Screenshot 2024-01-16 at 13 26 22" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/d0ae9e66-469f-4d3e-9e7b-a33e2b471fe4">

We should make further improvements to how we communicate the deposit schedule on the deposits list screen by displaying the same wording as the Payments Overview screen deposit schedule message.

**Comment by rogermattic** [comment]
Hey @Jinksi ! Thanks for creating this issue :) 

It would be great if we could keep the logic of how this notice behaves, however, change the design so it's rendered inline in that table, with this copy. This is how it might look like:

It would be great if we could change the design and the copy of this notice to be rendered inline in the table with the same logic. Here's a mockup:

<img width="1440" alt="Deposits" src="https://github.com/Automattic/woocommerce-payments/assets/94967682/a2068f49-6540-4c18-8234-df253dd300c1">

The copy should read: 

> Your next deposit is scheduled for January 19, 2024. You can change your deposit schedule in Payments Settings.

Let me know what you think!

**Comment by Jinksi** [comment]
Looks great, thanks @rogermattic! I've updated the issue description with your new design.

**Comment by haszari** [comment]
Note this issue is dependent on what we decide for calculating the next payout date. See this issue for details: 

- #8146 

That issue is blocked while we discuss the risks/benefits of calculating payout date. Adding blocked here, as we'll want this message to use consistent wording / details.

**Comment by Jinksi** [comment]
This is now unblocked since we have an outcome on the wording of this for Payments → Overview in #8184 – we're not showing an estimated date.

**Comment by haszari** [comment]
Grabbing this since it would be good to get this done and shipped alongside #8184 in 7.3.

**Comment by haszari** [comment]
Splitting this into two issues, as we need to refactor to get this notice into the table.

- This issue #8026 – ensure the existing notice uses exact same wording / component / behaviour as payments overview
- #8260 

#8260 will requires refactor of the table, and likely update of `@woocommerce/components`. Splitting off to separate issue to unblock making the existing notice consistent & not show estimated payout date.
